src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -174,16 +186,28 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         overlays['Bike Share Locations'] = overlaysControl.bikeShareOverlay();
         overlays['Bike Routes'] = overlaysControl.bikeRoutesOverlay(map);
 
+        // TODO: handle hiding layers on home view more cleanly.
+        // Would be better to initialize but not add to map, to avoid flashing,
+        // although it probably won't be visible due to centering of home page text.
+        if (homepage) {
+            overlays['Bike Share Locations'].eachLayer(function(layer) { layer.hide(); });
+            overlays['Bike Routes'].eachLayer(function(layer) { layer.hide(); });
+        }
```

Comment:
Good point. If someone had them selected and went back to the home view, I suppose we would need to deal with that.